lib/customer.rb

```diff
@@ -21,9 +21,9 @@ def find_all_invoices_by_customer_id(id)
 
   def transactions
    invoices.flat_map(&:transactions)
-  end
+ end
 
-  def favorite_merchant
-    invoices.flat_map(&:merchants).max_by { |invoices| invoices(&:successful_charge?) }
-  end
+ def favorite_merchant
+  invoices.flat_map(&:merchants).max_by { |invoices| invoices(&:successful_charge?) }
```

Comment:
Unused block argument - `invoices`. You can omit the argument if you don't care about it.<br>Line is too long. [85/80]<br>Use 2 (not 1) spaces for indentation.